src/main/resources/db/changelog/changes/03-add-isDeleted-column-to-users-table.yaml

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-isDeleted-column-to-users
+      author: Ki_ne_rei_ko
+      changes:
+        - addColumn:
+            tableName: users
+            columns:
+              - column:
+                  name: isDeleted
```

Comment:
add this column in existing changeset 02
addColumn doesn't support auto rollback, so this could cause problems